packages/node_modules/@webex/plugin-meetings/src/statsAnalyzer/index.js

```diff
@@ -0,0 +1,344 @@
+    }
+  };
+
+
+  inboundrtp = (result, type) => {
+    if (!result) {
+      return;
+    }
+    if (result.type !== 'inbound-rtp') return;
+    const mediaType = type || 'audio';
+    const sendrecvType = 'recv';
+
+    if (!sendrecvType) return;
+
+    if (result.bytesReceived) {
+      let kilobytes = 0;
+
+      if (!this.statsResults.internal[mediaType][sendrecvType].prevBytesReceived) {
+        this.statsResults.internal[mediaType][sendrecvType].prevBytesReceived = result.bytesReceived;
+      }
+
+      if (!this.statsResults.internal[mediaType][sendrecvType].pliCount) {
+        this.statsResults.internal[mediaType][sendrecvType].pliCount = result.pliCount;
+      }
+
+      if (!this.statsResults.internal[mediaType][sendrecvType].packetsLost) {
+        this.statsResults.internal[mediaType][sendrecvType].packetsLost = result.packetsLost;
+      }
+
+      if (!this.statsResults.internal[mediaType][sendrecvType].packetsReceived) {
+        this.statsResults.internal[mediaType][sendrecvType].packetsReceived = result.packetsReceived;
+      }
+
+      if (!this.statsResults.internal[mediaType][sendrecvType].lastPacketReceivedTimestamp) {
+        this.statsResults.internal[mediaType][sendrecvType].lastPacketReceivedTimestamp = result.lastPacketReceivedTimestamp;
+      }
+
+      if (!this.statsResults.internal[mediaType][sendrecvType].nackCount) {
+        this.statsResults.internal[mediaType][sendrecvType].nackCount = result.nackCount;
+      }
+
+      const bytes = result.bytesReceived - (this.statsResults.internal[mediaType][sendrecvType].prevBytesReceived);
+
+      this.statsResults.internal[mediaType][sendrecvType].prevBytesReceived = result.bytesReceived;
+
+      kilobytes = bytes / 1024;
+      this.statsResults[mediaType][sendrecvType].availableBandwidth = kilobytes.toFixed(1);
+      this.statsResults[mediaType].bytesReceived = kilobytes.toFixed(1);
+
+      this.statsResults[mediaType][sendrecvType].pliCount = result.pliCount - this.statsResults.internal[mediaType][sendrecvType].pliCount;
+      this.statsResults[mediaType][sendrecvType].currentPacketsLost = result.packetsLost - this.statsResults.internal[mediaType][sendrecvType].packetsLost;
+      if (this.statsResults[mediaType][sendrecvType].packetsLost < 0) {
+        this.statsResults[mediaType][sendrecvType].packetsLost = 0;
+      }
+
+
+      this.statsResults[mediaType][sendrecvType].totalPacketsReceived = result.packetsReceived;
+      this.statsResults[mediaType][sendrecvType].packetsReceived = result.packetsReceived - this.statsResults.internal[mediaType][sendrecvType].packetsReceived;
+      this.statsResults.internal[mediaType][sendrecvType].packetsReceived = result.packetsReceived;
+
+      if (this.statsResults.internal[mediaType][sendrecvType].packetsReceived === 0) {
+        console.log(`No packests received for ${mediaType} `, this.statsResults.internal[mediaType][sendrecvType].packetsReceived);
```

Comment:
Can you use the logger proxy as used throughout the rest of the plugin for logging?